src/EA.Weee.Domain/Lookup/WeeeCategory.cs

```diff
@@ -19,7 +19,7 @@ public enum WeeeCategory
         [Display(Name = "Lighting equipment")]
         LightingEquipment = 5,
 
-        [Display(Name = "Electrical and electronic tools")]
+        [Display(Name = "electrical and electronic tools")]
```

Comment:
I don't think this one should be changed as it should be capitalised at the start of the sentence?

Reply:
restored